src/main/java/com/amazonaws/kinesisvideo/producer/ClientInfo.java

```diff
@@ -71,4 +82,16 @@ public int getLoggerLogLevel() {
     public boolean getLogMetric() {
         return mLogMetric;
     }
+
+    public int getAutomaticStreamingFlags() {
+        return mAutomaticStreamingFlags;
+    }
+
+    public void setAutomaticStreamingFlags(AUTOMATIC_STREAMING_FLAGS flag) {
+        if (flag == AUTOMATIC_STREAMING_FLAGS.AUTOMATIC_STREAMING_ALWAYS_CONTINUOUS) {
+            mAutomaticStreamingFlags = AUTOMATIC_STREAMING_ALWAYS_CONTINUOUS;
+        } else {
+            mAutomaticStreamingFlags = AUTOMATIC_STREAMING_INTERMITTENT_PRODUCER;
```

Comment:
any harm in setting it up in 32 and skip default here?

Reply:
Moved this logic inside the constructor